client/app/views/home_application.coffee

```diff
@@ -5,7 +5,7 @@ Modal = require './error_modal'
 
 # Row displaying application name and attributes
 module.exports = class ApplicationRow extends BaseView
-    className: "application w360-33 w640-25 full-20 mod left"
+    className: "application w360-33 w640-25 full-20 mod"
```

Comment:
@poupotte can you look at https://github.com/cozy/cozy-home/pull/600/files and see if there's a conflict?